### DISCUSSION TITLE
[blocks]: toolbar

### DISCUSSION THREAD

**Discussion by Elaskin** [shadcn-ui/ui#3339] [Blocks Request]
### Description

A toolbar that collapses all buttons that didn't fit on the sreecn to some kind of dropdown menu.

### Example

https://js.devexpress.com/React/Demos/WidgetsGallery/Demo/Toolbar/Adaptability/MaterialBlueLight/

**Comment by horiuchi** [comment]
I want to add the ToolBar component too.

https://github.com/shadcn-ui/ui/issues/2633
This issue is also related.
